scripts/check_pr_size.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+MAX_LINES=400
```

Comment:
It can be usefull to handle this value from within Github environment variables. Thay way, if we need to fine tune it we don't need to change the script. What do you think?